docker-compose.yml

```diff
@@ -26,15 +26,37 @@ services:
       POSTGRES_USER: postgres
       POSTGRES_DB: datastore_local
 
-  runner-app:
-    container_name: runner-app
+  forms-email-app:
```

Comment:
As with `fb-component-acceptance-tests` there isn't a single `runner-app` for _all_ specs